File: CarsMatterApp/app/src/main/java/com/andrey/carsmatter/models/KindOfService.java
package com.andrey.carsmatter.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class KindOfService {
    @SerializedName("id")
    @Expose
    public String Id;

    @SerializedName("name")
    @Expose
    public String Name;
}

File: CarsMatter.Infrastructure/Models/MsSQL/KindOfService.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace CarsMatter.Infrastructure.Models.MsSQL
{
    public class KindOfService
    {
        public string Id { get; set; }

        public string Name { get; set; }

        [JsonIgnore]
        public ICollection<ConsumablesNote> ConsumablesNotes { get; set; }

        public KindOfService()
        {
            this.ConsumablesNotes = new List<ConsumablesNote>();
        }
    }
}
